plugins/woocommerce-blocks/assets/js/blocks/product-collection/collections/README.md

```diff
@@ -0,0 +1,47 @@
+# Product Collection - Collections
+
+_Note: Collections documented here are internal implementation. It's not a public API._
```

Comment:
Even though I agree with the overall intention to enable 3PD to register collections in near future, most likely it won't be through this file and the way it's described in the file. So adding `yet` may give a wrong impression that this will be the way while it won't 🤔 

I'll try to rephrase that so it says we'd like to allow for custom Collection registration in the future but not yet and not like here. Good call!